pkg/cli/generic_flag.go

```diff
@@ -199,31 +210,31 @@ func (val *genericType[T]) Set(str string) error {
 	case *bool:
 		v, err := strconv.ParseBool(str)
 		if err != nil {
-			return errors.Errorf("error parse: %w", err)
+			return errors.Errorf(`must be one of: "0", "1", "f", "t", "false", "true"`)
 		}
 
 		*dest = v
 
 	case *int:
 		v, err := strconv.ParseInt(str, 0, strconv.IntSize)
 		if err != nil {
-			return errors.Errorf("error parse: %w", err)
+			return errors.Errorf("must be 32-bit integer")
 		}
 
 		*dest = int(v)
 
 	case *uint:
 		v, err := strconv.ParseUint(str, 10, 64)
 		if err != nil {
-			return errors.Errorf("error parse: %w", err)
+			return errors.Errorf("must be 32-bit unsigned integer")
```

Comment:
Why aren't you wrapping the error? It's generally a good practice to.

Reply:
Explained above.